models.py

```diff
@@ -5,8 +5,16 @@ def __init__(self, name, hp=100.0, strength=0.0, defence=1.0):
         self.strength = strength
         self.defence = defence
 
+    def attack_strenght(self,strength):
+        import random
```

Comment:
All the imports should be always on the top of the file.
https://stackoverflow.com/a/128577/12030524